src/cli/argument-parser.js

```diff
@@ -124,8 +138,9 @@ export default class CLIArgumentParser {
     _parseFilteringOptions () {
         this.opts.testGrep    = CLIArgumentParser._optionValueToRegExp('--test-grep', this.opts.testGrep);
         this.opts.fixtureGrep = CLIArgumentParser._optionValueToRegExp('--fixture-grep', this.opts.fixtureGrep);
+        this.opts.meta = CLIArgumentParser._optionValueToJson('--meta', this.opts.meta);
 
-        this.filter = (testName, fixtureName) => {
+        this.filter = (testName, fixtureName, _, meta) => {
```

Comment:
Do we need `fixturePath` parameter in this method? As I can see we do not use nowhere

Reply:
The `filter` callback in the `Runner` class passes the [`fixturePath`](http://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#filter) as the 3rd argument.

Although, the `fixturePath` argument isn't used in for filtering in the cli, the api defines the callback with that argument, i won't suggest to change the api, it might break people's code.

Reply:
You're right. https://github.com/DevExpress/testcafe/blob/fa295f84a8c3047576216430b73729c3684e7d96/src/cli/cli.js#L95